com.unity.ml-agents/Runtime/SideChannels/EnvironmentParametersChannel.cs

```diff
@@ -42,12 +45,17 @@ protected override void OnMessageReceived(IncomingMessage msg)
             {
                 var value = msg.ReadFloat32();
 
-                m_Parameters[key] = value;
+                m_Parameters[key] = () => value;
 
```

Comment:
When floats are sent from python, callbacks are triggered as per conversation in the design review.  When sampler configs are sent, they are not.